modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -12,20 +12,87 @@ PyObject* pyopencv_from(const std::vector<GCompileArg>& value)
     return pyopencv_from_generic_vec(value);
 }
 
+template<>
+bool pyopencv_to(PyObject* obj, GRunArgs& value, const ArgInfo& info)
+{
+    return pyopencv_to_generic_vec(obj, value, info);
+}
+
+static PyObject* from_grunarg(const GRunArg& v)
+{
+    switch (v.index())
+    {
+        case GRunArg::index_of<cv::Mat>():
+        {
+            const auto& m = util::get<cv::Mat>(v);
+            return pyopencv_from(m);
+        }
+
+        case GRunArg::index_of<cv::Scalar>():
+        {
+            const auto& s = util::get<cv::Scalar>(v);
+            return pyopencv_from(s);
+        }
+
+        default:
+            return NULL;
```

Comment:
Copy-paste from `opencv`, make sense to use `nullptr`, but let it be implemented in next PR's to minimize diff :)